forte_wrapper/nltk/nltk_processors.py

```diff
@@ -68,6 +78,27 @@ def _process(self, input_pack: DataPack):
         for token, tag in zip(token_entries, taggings):
             token.pos = tag[1]
 
+    def record(self, record_meta: Dict[str, Set[str]]):
+        r"""Method to add output type record of current processor
+        to :attr:`forte.data.data_pack.Meta.record`.
+
+        Args:
+            record_meta: the field in the datapack for type record that need to
+                fill in for consistency checking.
+        """
+        record_meta["ft.onto.base_ontology.Token"] = {"pos"}
+
+    @classmethod
+    def expected_types_and_attributes(cls):
+        r"""Method to add expected type for current processor input which
```

Comment:
Most of time, the overwrite methods can write the specific behavior and keep the repeated description that appear in the parent class brief. Here, it'd better mention what types are expected.